docs/source/examples.rst

```diff
@@ -4,6 +4,8 @@
 Examples
 =========
 
+Note: We are working on new building blocks and datasets. Some of the components in the examples (e.g. Field) will eventually retire. 
```

Comment:
Does this file give enough information about older datasets?

Reply:
refer to release note.